xonsh/tools.py

```diff
@@ -228,6 +228,30 @@ def __radd__(self, other):
             other = other._l
         return EnvPath(other + self._l)
 
+    def add(self, data, toStart=False, replace=False):
+        """
+        add a value to this EnvPath,
```

Comment:
Can you please conform to [the numpydoc style](http://numpydoc.readthedocs.io/en/latest/format.html) for docstrings. Here is an example: https://github.com/xonsh/xonsh/blob/master/xonsh/tools.py#L577